packages/slice-machine/server/src/api/services/getEnv.ts

```diff
@@ -65,49 +62,9 @@ function extractRepo(parsedRepo: ParseResult): string | undefined {
   }
 }
 
-function handleBranch(): Promise<{ branch?: string; err?: Error }> {
```

Comment:
i'm not sure what this does.

Reply:
Checking some ref in github for chromatic. But as chromatic is not used anymore, we should remove this function

Reply:
if so, remove also all references of chromatic, in the manifest, env etc